model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -79,7 +79,7 @@ class ExcludesTest : WordSpec() {
             "return true if all occurrences of the package are excluded" {
                 val excludes = Excludes(
                         projects = listOf(ProjectExclude("path1", ProjectExcludeReason.BUILD_TOOL_OF, "comment")),
-                        scopes = listOf(ScopeExclude("testScope", ScopeExcludeReason.PROVIDED_BY, "comment"))
+                        scopes = listOf(ScopeExclude(Regex("testScope"), ScopeExcludeReason.PROVIDED_BY, "comment"))
```

Comment:
Did you consider to provide a constructor for ScopeExclude to take a String which internally creates a regex?

Reply:
Done.